src/arrayMethodSort.js

```diff
@@ -3,9 +3,40 @@
 /**
  * Implement method Sort
  */
+
+const compareAsSrings = (a, b) => {
+  const stringA = String(a);
+  const stringB = String(b);
+
+  if (stringA > stringB) {
+    return 1;
+  } else if (stringA === stringB) {
+    return 0;
+  }
```

Comment:
When `return` is written inside `if` , `else` is redundant.
```suggestion
  if (stringA > stringB) {
    return 1;
  } 
  
  if (stringA === stringB) {
    return 0;
  }
```